tests/integration/integration.json

```diff
@@ -15,6 +15,14 @@
         "tests/data/reference/bwa/reference.fasta.pac"
     ]
   },
-  "SVcalling.dockerImagesFile": "dockerImages.yml"
+  "SVcalling.dockerImages": {
```

Comment:
I would use the defaults here. (I.e. not specify a dockerimages input). Otherwise you will have to update the defaults in two places.

Reply:
True. Let me change that